src/manifest.c

```diff
@@ -457,12 +457,13 @@ static int try_delta_manifest_download(int current, int new, char *component, st
 			unlink(deltafile);
 			goto out;
 		}
-
+/*
 		if (!download_and_verify_signature(url, deltafile)) {
 			ret = -1;
 			unlink(deltafile);
 			goto out;
 		}
+*/
```

Comment:
Unrelated to this patch? I would suggest to remove code entirely if it's not needed at present and won't be in the future.